invenio_banners/resources/config.py

```diff
@@ -18,7 +21,21 @@ class BannerResourceConfig(RecordResourceConfig):
     # Blueprint configuration
     blueprint_name = "banners"
     url_prefix = "/banners"
-    routes = {"create": "/new"}
+    routes = {
+        "create": "/new",
+        "banner": "/<banner_id>",
+        "list": "/",
+        "active": "",
```

Comment:
is this route needed? I think "active" it is only an url argmuent, but the route is "list"